### PR TITLE
Added coder.com

### PR DESCRIPTION
**Pull request by annnoo**
Added _Coder_

Coder is a platform that gives you a whole development environment build around Visual Studio Code. In the free tier you will get 2GB of project and 2 GB of container space as well as 5 hours of Fast Time that dynamically scales your container up if enabled.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -565,7 +565,7 @@ Table of Contents
    * [codesandbox.io](https://codesandbox.io/) — Online React playground
    * [stackblitz.com](https://stackblitz.com/) — Online VS Code IDE for Angular & React
    * [cacher.io](https://www.cacher.io) - Code snippet organizer with labels and support for 100+ programming languages
-
+   * [Coder](https://coder.com) - Online development environment built around VSCode
```
Comment (ripienaar) [inline README.md]: unfortunately there appears no indication of price or what eventual priving or free tier would look like, its just a alpha release.

Great, but I am not going to add something that on release might have no free tiers - so if there's a link I missed on their site that outlines please let me know, we should then update the description here to capture the details of whats available for free

thanks!

Comment (annnoo) [reply README.md]: After looking through their homepage I just noticed that their current pricing model is only described if you are logged in:

https://coder.com/store

image

Comment (ripienaar) [reply README.md]: hmm, ok, mind updating the description with some version of that? Mention the plans are visible once logged in

Comment (annnoo) [reply README.md]: Hope this is okay :)

Comment (ripienaar) [reply README.md]: no update :)

Comment (annnoo) [reply README.md]: My bad ... ^^ 
I only edited it here in the pull request. Should be fine now!

**Comment by ripienaar** [comment]
thank you for sticking with it :)

**Pull request merged**